src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
```

Comment:
It is a good practice to avoid using pixel values directly. Consider using variables for commonly used values. For example, you can define a variable `$padding: 16px;` at the top of your SCSS file and use it instead of hard-coded pixel values. This makes your code more maintainable and easier to read.